microsetta_interface/templates/new_participant.jinja2

```diff
@@ -0,0 +1,255 @@
+      var valid = min_validation("13-17-form");
+      form = document.getElementById("13-17-form");
+      if(form.consent_parent.checked == false) {
+        form.consent_parent.className += " highlight";
+        valid = false;
+      }
+      if(form.consent_child.checked == false) {
+        form.consent_child.className += " highlight";
+        valid = false;
+      }
+      if(form.parent_1_name.value.trim().length == 0) {
+        form.parent_1_name.className += " highlight";
+        valid = false;
+      }
+      return valid;
+    }
+
+    function change_consent_div(radio) {
+        $("#0-6-div").hide();
+        $("#7-12-div").hide();
+        $("#13-17-div").hide();
+        $("#18-plus-div").hide();
+        $("#"+ radio.value +"-div").show();
+    }
+
+    function relax_parents(box) {
+        var sec_parent = $("#" + box.form.id + "_parent_2_name");
+        if(box.checked) {
+            sec_parent.required = false;
+        } else {
+            sec_parent.required = true;
+        }
+    }
+
+    $(document).ready(function() {
+        $("#0-6-div").hide();
+        $("#7-12-div").hide();
+        $("#13-17-div").hide();
+        $("#18-plus-div").hide();
+    });
+</script>
+<style>
+.scrolltext {
+  height: 400px;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  width: 80%;
+  margin: 0 auto;
+}
+</style>
+{% endblock %}
+{% block breadcrumb %}
+    <li class="breadcrumb-item active" aria-current="page">Consent</li>
+{% endblock %}
+{% block content%}
+<h2>{{ tl['ADD_HUMAN_TITLE'] |e}}</h2>
+{% if message %}
+<h3 style="color: red">{{message |e}}</h3>
+{% endif %}
+<div id="age-ranges-div" style='text-align: left;padding: 10px'>
+    <h3>{{ tl['SEL_AGE_RANGE'] |e}}</h3>
+    <input type='radio' id='age-range-1' name='age-range' value='0-6' onchange="change_consent_div(this)"> <label for='age-range-1'>{{ tl['AGE_0_6'] |e}}</label><br />
+    <input type='radio' id='age-range-2' name='age-range' value='7-12' onchange="change_consent_div(this)"> <label for='age-range-2'>{{ tl['AGE_7_12'] |e}}</label><br />
+    <input type='radio' id='age-range-3' name='age-range' value='13-17' onchange="change_consent_div(this)"> <label for='age-range-3'>{{ tl['AGE_13_17'] |e}}</label><br />
+    <input type='radio' id='age-range-4' name='age-range' value='18-plus' onchange="change_consent_div(this)"> <label for='age-range-4'>{{ tl['AGE_18'] |e}}</label>
+    <hr>
+</div>
+
+<div id="0-6-div" style='text-align: left;padding: 10px'>
+<div class='scrolltext'>
+{{ tl['CONSENT_YOUR_CHILD'] }}
+{{ tl['PARTICIPATION_AGREEMENT'] }}
+{{ tl['EXHIBIT_A'] }}
+</div>
+    <div class="" id="consent">
+        <hr>
+        <a href="http://oag.ca.gov/sites/all/files/agweb/pdfs/research/bill_of_rights.pdf" target="_blank">{{ tl['BILL_OF_RIGHTS']|e}}</a><br/>
+        <form id="0-6-form" name="consent_info" onsubmit="return validate06();" action="{{ post_url }}" method="post">
```

Comment:
Where is {{ post_url }} getting passed in?  It looks like there were changes made during the migration that might have stopped sending this in, only ting I see being passed to jinja2 renderer is the variable tl and the default variables passed to all templates.  Do the post_url links work in the consent doc after the migration?  There should be several of them in this file.

Reply:
Zero changes with what was in microsetta-private-api:

```bash
(microsetta-admin-37) 13:51:26 (dtmcdonald@here):tr$ wget https://raw.githubusercontent.com/biocore/microsetta-private-api/master/microsetta_private_api/templates/new_participant.jinja2
--2020-07-08 13:51:28--  https://raw.githubusercontent.com/biocore/microsetta-private-api/master/microsetta_private_api/templates/new_participant.jinja2
Resolving raw.githubusercontent.com... 151.101.196.133
Connecting to raw.githubusercontent.com|151.101.196.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 13040 (13K) [text/plain]
Saving to: 'new_participant.jinja2'

new_participant.jinja2                            100%[=============================================================================================================>]  12.73K  --.-KB/s    in 0.04s

2020-07-08 13:51:29 (344 KB/s) - 'new_participant.jinja2' saved [13040/13040]

(microsetta-admin-37) 13:51:29 (dtmcdonald@here):tr$ diff new_participant.jinja2 microsetta-interface/microsetta_interface/templates/new_participant.jinja2
(microsetta-admin-37) 13:51:29 (dtmcdonald@here):tr$
```

Reply:
post_url is coming from the private API in the json object

Reply:
Change is in the set of variables passed in to jinja2 rendering:

``` 
    consent_html = render_template(
        "new_participant.jinja2",
        tl=localization_info[localization.NEW_PARTICIPANT_KEY],
        lang_tag=language_tag,
        post_url=consent_post_url
    )
```

```
    return _render_with_defaults('new_participant.jinja2',
                                 tl=consent_output)
```

Reply:
ah, I see now. the page "works" in that i could advance but ill verify the url is there